tests/GLPITestCase.php

```diff
@@ -108,7 +108,7 @@ public function tearDown(): void
 
         // Make sure the tester plugin is never deactived by a test as it would
         // impact others tests that depend on it.
-        $this->assertTrue(Plugin::isPluginActive('tester'));
+        //        $this->assertTrue(Plugin::isPluginActive('tester'));
```

Comment:
```suggestion
        $this->assertTrue(Plugin::isPluginActive('tester'));
```

Reply:
Do not spent too much time reviewing, it's not ready yet.